### PR TITLE
[misc] Add a link to the website, update user count, update .gitignore

### PR DESCRIPTION
**Pull request by payne911**
### Purpose of pull request

* In the `README`: added a link to the website, and fix a typo
* Update user count
* Update `.gitignore`

### Pull request checklist

- [ ] Branch and PR are named correctly
- [x] Updated relevant documentation
- [ ] Tested locally on the website
- [ ] Wrote unit tests for changes

### Testing instructions

N/A

### CODE REVIEW

**Comment by nikgil** [comment]
If you could change the body and header to similar format of: https://github.com/cscareerhub/csch-web/pull/19
that would be helpful (this automates tickets closing)

**Comment by payne911** [comment]
But this doesn't relate to any issue.

Unless a PR is a ticket?

**Comment by nikgil** [comment]
still better to maintain consistency in format

**Comment by payne911** [comment]
Done :)

**Review by nikgil — APPROVED** [review]

**Comment by payne911** [comment]
Hum, I just realized: am I supposed to get this PR to target the `stage` branch instead of the `master` ?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We are a community of software engineers on a mission of lifelong learning.
 
 CS Career Hub was created to help guide engineers at any point in their career. Over 18,000 users have joined our community, ranging from first-year students to hiring managers. In the past few years, we've facilitated thousands of mock interviews and resume reviews with candidates. With plenty of plans on the roadmap, you can expect even more to come.
```
Comment (brendacs) [inline README.md:7]: can you also update this to `20,000 users`?

**Comment by payne911** [comment]
I just realized this whole sentence could be filled with various hyperlinks:

> You'll find information about CSCH, the staff that runs it, links to the Wiki & FAQ, and ways to contact us.

**Comment by brendacs** [comment]
> I just realized this whole sentence could be filled with various hyperlinks:
> 
> 
> 
> > You'll find information about CSCH, the staff that runs it, links to the Wiki & FAQ, and ways to contact us.

Not sure it's necessary since the website's front page already has all those links. That seems to be overdoing it.

**Review by brendacs — APPROVED** [review]

**Comment by payne911** [comment]
@brendacs just making sure because you approved right in the middle of my `git push`, apparently: are you fine with me adding a few things to that `.gitignore` ?

File: .gitignore [hunk 2]
```diff
@@ -29,4 +29,9 @@ yarn-error.log*
...
 .env.production.local
 
-*.d.ts
\ No newline at end of file
+*.d.ts
+
+# intelliJ
+.idea/**
+*.iml
+*.ipr
```
Comment (brendacs) [inline .gitignore:37]: newline

**Comment by brendacs** [comment]
@payne911 one comment but otherwise, looks good

File: .gitignore (hunk 2 above)
Comment (payne911) [reply .gitignore]: Just pushed it. Mind explaining why?

**Comment by payne911** [comment]
Alright, and please note I changed the target of this PR to `stage`.

**Comment by brendacs** [comment]
one last thing is to just make sure to "squash and merge" once you're ready to merge

**Comment by payne911** [comment]
> one last thing is to just make sure to "squash and merge" once you're ready to merge

I do not have write access into this repository, though. Unless I am misunderstanding what you mean?

**Comment by brendacs** [comment]
> > one last thing is to just make sure to "squash and merge" once you're ready to merge
> 
> I do not have write access to this repository, though. Unless I am misunderstanding what you mean?

Oops, I forgot

File: .gitignore (hunk 2 above)
Comment (brendacs) [reply .gitignore]: @payne911, this explains it better than I would: https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

**Pull request merged**